public/src/cards.js

```diff
@@ -140,7 +139,7 @@ let events = {
       });
     }
 
-    _.download(data.join("\n"), filename);
+    _.download(data.join("\n"), `${filename}-draftlog.txt`);
```

Comment:
Did you change the file extension to `.txt` for a particular reason?

`.txt` and `.log` both work on https://magicprotools.com/draft/upload

Reply:
@HerveH44

Reply:
Ehh, i just imagined that it woul be clear that it's a txt. No particular reason no!